WooCommerce/Classes/ViewRelated/Orders/Simple Payments/Amount/SimplePaymentsAmountViewModel.swift

```diff
@@ -190,10 +194,10 @@ private extension SimplePaymentsAmountViewModel {
     }
 }
 
-// MARK: Definitions
-extension SimplePaymentsAmountViewModel {
-    /// Representation of possible notices that can be displayed
-    enum Notice: Equatable {
-        case error
+// MARK: Constants
+private extension SimplePaymentsAmountViewModel {
+    enum Localization {
+        static let creationError = NSLocalizedString("There was an error creating the order",
+                                                     comment: "Notice text after failing to create a simple payments order.")
```

Comment:
nit: Since we're using this string for the error now, we can remove the `error` string from the `Localization` enum in `SimplePaymentsAmount.swift`:

https://github.com/woocommerce/woocommerce-ios/blob/18389169a5750ea006fd88fb7573da53b222a0e5/WooCommerce/Classes/ViewRelated/Orders/Simple%20Payments/Amount/SimplePaymentsAmount.swift#L149

Reply:
Ups, yeah forgot to do that! nice catch!